src/components/Settings/Export/Export.helpers.js

```diff
@@ -781,6 +802,97 @@ export async function pdfExportAdapter(boards = [], intl) {
   }
 }
 
+export async function picseePdfExportAdapter(boards = [], intl) {
```

Comment:
Can you refactor the `pdfExportAdapter` function to avoid innecesary code repetition? Note that the function is also used in `PrintBoardButton.container.js`

Reply:
I have refactored `pdfExportAdapter` in my newest commit to remove unnecessary code repetition.